### ISSUE TITLE
Try and run cashflowy on t2.micro. Currently it runs on t2.small

### ISSUE THREAD

**Issue by alexjv89** [asyncauto/cashflowy#38]
It should be possible to run cashflowy on t2.micro. Currently it runs on t2.small. This can lead to some savings in cost.

**Comment by aptro** [comment]
This is feasible now. But once we add package like aws-sdk which takes significant memory to build, will break in t2.micro.

**Comment by alexjv89** [comment]
Do we want aws-sdk now? 

Lets pay for it when we use it. Right now it sort of makes sense to be very frugal and to keep costs low

**Comment by aptro** [comment]
Yup. For s3 operations. If we able to find any simple npm package that does s3 cruds then can move to  t2.micro.
